dune

```diff
@@ -1,14 +1 @@
-(subdir
- node_modules
- (dirs reason-react)
- (data_only_dirs reason-react)
- (subdir
-  reason-react/src
-  (include_subdirs unqualified)
-  (library
-   (name reason_react)
-   (wrapped false)
-   (modes melange)
-   (flags -w -9-27-32-20)
-   (preprocess
-    (pps melange.ppx reactjs-jsx-ppx)))))
+(dirs src)
```

Comment:
There will be a `node_modules` folder, which might grow quite a lot for real-world projects. Ignoring that folder by not adding it to the whitelist can help Dune run faster by avoiding some wasteful traversal work.

As a "template" this project should show good practices. Assuming this is one 😄 if it might be confusing, maybe I can add a comment explaining why it is better to whitelist the source folders?

Reply:
how about we do it the other way instead?

```lisp
;; we start by ignoring node_modules. if you need to consume an OCaml 
;; project from `node_modules`, this should work:
;; - remove the `dirs` stanza below
;; - add a `(subdir node_modules (dirs only_your_package))`
(dirs :standard \ node_modules)

Reply:
Sounds good, added in https://github.com/melange-re/melange-opam-template/pull/13/commits/56c3519bf7c98872f7d90f953afd7df1ac6b2d9e.